spec/features_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'takeaway'
+require 'menu'
+require 'order'
+
+# describe 'User stories + feature tests: ' do
+
+describe 'User Stories' do 
+
+# As a customer
+# So that I can check if I want to order something
+# I would like to see a list of dishes with prices
+
+	describe '#read menu' do
+
+	subject(:menu) {Menu.new(dishes)}
+
+	let(:dishes) do
+		{
+		chicken: 3.99,
+		falafel: 4.50
+		}
+	end
+
+	it 'enables a customer to view a list of dishes with prices' do
+		expect(menu.dishes).to eq(dishes)
+	end
+end
```

Comment:
Lint/BlockAlignment: `end` at 27, 0 is not aligned with `describe '#read menu' do` at 13, 1.